tests/js/utils.js

```diff
@@ -463,15 +463,17 @@ export const provideKeyMetricsUserInputSettings = (
  * Provides notifications data to the given registry.
  *
  * @since 1.140.0
+ * @since n.e.x.t Updated the `overwrite` option to be a named parameter.
  *
- * @param {Object}  registry    The registry to set up.
- * @param {Object}  [extraData] Extra data to merge with the default settings.
- * @param {boolean} overwrite   Merges extra data with default notifications when false, else overwrites default notifications.
+ * @param {Object}  registry            The registry to set up.
+ * @param {Object}  [extraData]         Extra data to merge with the default settings.
+ * @param {Object}  [options]           Options object.
+ * @param {boolean} [options.overwrite] Merges extra data with default notifications when false, else overwrites default notifications.
  */
 export const provideNotifications = (
 	registry,
 	extraData,
-	overwrite = false
+	{ overwrite = false } = {}
```

Comment:
It's preferable to avoid "unnamed" boolean parameters as they are not very clear when in use.

```js
			provideNotifications(
				registry,
				{
					[ FPM_SETUP_CTA_BANNER_NOTIFICATION ]:
						DEFAULT_NOTIFICATIONS[
							FPM_SETUP_CTA_BANNER_NOTIFICATION
						],
				},
				true, // What does `true` mean?
			);
```